docs-chef-io/content/inspec/license.md

```diff
@@ -211,3 +211,14 @@ inspec exec <PROFILE_NAME>
 ```
 
 This capability is basic and you must synchronize the license servers, otherwise you may get inconsistent results.
+
+
+## Chef's licensing telemetry
+
+Chef Licensing telemetry service gathers product activation, usage and/or environment information, identify trends and/or bugs, collect usage statistics, and/or track other data related to your use of product.
+
+Currently this feature is enabled for free and trial tiers only. This feature collects product usage data to help Chef better serve its users.
+
+Key Points:
+- **Data Collection**:  The telemetry feature gathers is as described in current version of privacy policy available at  https://www.progress.com/legal/privacy-policy.
```

Comment:
`The telemetry feature gathers is as described in current version of privacy policy available at` Does this line missing `data` word after gathers?

Reply:
Yeah it should be better edited for language too.